src/components/WebexVideoSettings/WebexVideoSettings.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import {Icon} from '@momentum-ui/react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import {WEBEX_COMPONENTS_CLASS_PREFIX} from '../../constants';
+import WebexLocalMedia from '../WebexLocalMedia/WebexLocalMedia';
+import WebexMeetingControl from '../WebexMeetingControl/WebexMeetingControl';
+import {useMeetingControl} from '../hooks';
+
+const noCameraMsg = 'Can\'t detect your camera. Make sure your camera is connected and try again.';
+
+/**
+ * Webex Video Settings component
+ *
+ * @param {object} props  Data passed to the component
+ * @param {string} props.meetingID  ID of the meeting from which to obtain local media
+ * @param {string} props.className  Custom CSS class to apply
+ * @returns {object} JSX of the component
+ */
+export default function WebexVideoSettings({className, meetingID}) {
+  const [, display] = useMeetingControl('switch-camera', meetingID);
```

Comment:
we should not need to look into switch-camera control's internals... let me think of an alternative